server_admin/topics/users/viewing.adoc

```diff
@@ -1,24 +1,22 @@
-[searching-users]
+[id="proc-searching-users_{context}"]
 = Searching for users
 
-You can search for a user in the database, by using the full name, last name, or email address of the user.
+Search for users to view their detailed information and to understand their role and responsibilities in the console. You can search for a user in the database using the full name, last name, first name, or email address of the user.
 
-.Prerequisites
-* You are logged in to the correct realm and are not in the master realm.
 
 .Procedure
-. Click on *Users* in the left menu bar. This will direct you to the user list page.  
-. In the search box, type the full name, last name, or email address of the user you want to search for in the user database. The query will bring up all users that match your criteria.  
+. Click *Users* in the left menu bar. This opens the user list page.  
+. In the search box, type the full name, last name, firstname, or email address of the user you want to search for in the user database. The serach returns all users that match your criteria.  
 . Alternatively, you can click *View all users* to list every user in the system.  
 +
-NOTE: This will only search the local {project_name} database and not the federated database, such as LDAP. The backends for federated databases, such as LDAP, do not have a pagination mechanism that allow for the scanning of users. 
+NOTE: This will only search the local {project_name} database and not the federated database, such as LDAP. The backends for federated databases do not have a pagination mechanism that enables searching for users. 
```

Comment:
I am slowly trying to teach developers to always follow "this" with a noun. So it should be "This action..."
"only search" should be "search only"

Reply:
@andymunro I don't think I understand what you mean by this comment.